### DISCUSSION TITLE
distributed-transaction/TCC

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#160] [General]
# distributed-transaction/TCC

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/distributed-transaction/TCC.html

**Comment by chainray01** [comment]
> 5.2分区容错性（Partition tolerance）：代表分布式环境中部分节点因网络原因而彼此失联后，即与其他节点形成“网络分区”时，系统仍能正确地提供服务的能力。x

x为错误敲入

**Reply by isno** [reply]
谢谢，已修正。
